Tests/Extensions/DictionaryNSAttributedStringKeySpec.swift

```diff
@@ -0,0 +1,22 @@
+@testable import ELDebateFramework
+import Nimble
+import Quick
+
+class DictionaryNSAttributedStringKeySpec: QuickSpec {
+
+    override func spec() {
+        describe("Dictionary") {
```

Comment:
First describe inside spec function should contain name of what is under test, and here is an extension `Dictionary+StringAny` so describe should looks like below:

```swift
...
describe("Dictionary+StringAny") {
...
```